src/ray/raylet/scheduling_policy.h

```diff
@@ -29,8 +29,9 @@ class SchedulingPolicy {
   ///         configured and current resource capacity on each node.
   /// \return Scheduling decision, mapping tasks to node managers for placement.
   std::unordered_map<TaskID, ClientID> Schedule(
```

Comment:
Can you make it clear in this documentation that `cluster_resources` will be modified to reflect the task placements?

Also, I think it would help to have an overview of the current scheduling policy here as well.